iceberg-common/pom.xml

```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  Copyright (c) 2025 NVIDIA CORPORATION.
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+     http://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>com.nvidia</groupId>
+        <artifactId>rapids-4-spark-jdk-profiles_2.12</artifactId>
+        <version>25.06.0-SNAPSHOT</version>
+        <relativePath>../jdk-profiles/pom.xml</relativePath>
+    </parent>
+
+    <artifactId>rapids-4-spark-iceberg-common_2.12</artifactId>
+    <name>Common module of RAPIDS Accelerator for Apache Iceberg</name>
+    <description>Apache Iceberg support for the RAPIDS Accelerator for Apache Spark common module</description>
+    <version>25.06.0-SNAPSHOT</version>
+
+    <properties>
+        <rapids.compressed.artifact>false</rapids.compressed.artifact>
+        <rapids.shim.jar.phase>package</rapids.shim.jar.phase>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>com.nvidia</groupId>
+            <artifactId>rapids-4-spark-sql_${scala.binary.version}</artifactId>
+            <version>${project.version}</version>
+            <classifier>${spark.version.classifier}</classifier>
```

Comment:
since this module consumes spark.version.classifer dependency and this module should produce a spark.version.classfier artifact. We can use this config  https://maven.apache.org/plugins/maven-shade-plugin/shade-mojo.html#shadedClassifierName to make a per-spark shim output